app/src/docker/docker-compose.yml

```diff
@@ -355,6 +359,9 @@ services:
         condition: service_started
       db-init:
         condition: service_started
+    volumes:
+      - pgdata:/persist/postgres
+      - tracking:/persist/tracking
```

Comment:
VRO Console needs access to that folder in order to load the payload for resubmitting the request.